go.mod

```diff
@@ -1,6 +1,6 @@
 module github.com/sigstore/sigstore
 
-go 1.17
+go 1.19
```

Comment:
this means we will only support go1.19, is that what we want? or keep that in go 1.18 for now

Reply:
I think most sigstore repositories were migrating to 1.19? Happy to leave at 1.18 and leave test strategies to 1.18 - 1.19 though, and we can make the decision to bump to 1.19 in a separate PR

Reply:
yes, but we keep the go modules in 1.18


i would like to always use the latest, but then we might broke downstream users 🙃

Reply:
AH! makes sense! updated!

Reply:
thank you!